docs/android/v2/migrations/surfaceview-migration.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: SurfaceViewRenderer to HMSVideoView
+nav: 14.1
+description: Migrating from SurfaceViewRenderer --> HMSVideoView
+---
+
+HMSVideoView provides a better abstarction to render live video, comes with additionaly features like zoom and pan, auto-bitrate adjuster on the playback side.
```

Comment:
Add more reasons on the motivation of migrating from SurfaceViewRender to HMSVideoView